### ISSUE TITLE
Turkish translate - font problem

### ISSUE THREAD

**Issue by yusufff** [ncase/trust#19]
Hi, I'm translating this and having problem with the font. As you can see here: https://yusufff.github.io/trust/
https://github.com/yusufff/trust

What can we do?

**Comment by osaatcioglu** [comment]
Yusuf, I am also translating to Turkish as I said in this thread https://github.com/ncase/trust/issues/5 We should collaborate. I see that you came pretty far as well. May be we can cross check our translations and share the rest. 

https://osaatcioglu.github.io/trust/

Regarding font, the font fits nicely with the overall design and Turkish characters don't look so bad. What do you see really?

**Comment by yusufff** [comment]
@osaatcioglu That's wonderfull, we can collaborate and win 2 coin 😉 My designer friend says she can convert the Futurama Handwritten font to Turkish. The "İ"s bugs me. I'll let you know when she's done and I'm finish with the translation.

**Comment by osaatcioglu** [comment]
:) It doesn't look like a big problem to me. But, if your friend can provide a better font, I am all in for it. :) The only problem I see is that the text is longer in Turkish, of course, and this requires some cutting here and there to make sure that all text is visible. 

Why wait till the end of translation? We can make better progress by collaborating earlier. What's your email address? Mine is osaatcioglu at gmail.

**Comment by CCanerT** [comment]
Hi, I am not very good at programming but I am good with photoshop and I can translate stuff. Anything you want me to handle?

**Comment by osaatcioglu** [comment]
I finished my translation and created a pull request (https://github.com/ncase/trust/pull/29). It will be great if you can do proof reading and open an issue in my repo. Thanks. 

https://osaatcioglu.github.io/trust/

**Comment by CCanerT** [comment]
There are some hard to understand parts, but I think it is because of the limitations of space provided by the game. Other than that, I didn't run into any errors.

**Comment by osaatcioglu** [comment]
I agree. I had to summarize some parts brutally to fit it to the space. If you can list them as an issue, I will take another look. Thanks @CCanerT

**Comment by jkoelling** [comment]
It's possible to gain some more room by changing the y coordinates of some elements. Also, the splash for the introduction text can be widened up a bit. This way I could get a translation into German without shortening the original much.

Have a look at the changes I made to the `slides/*.js` files: https://github.com/jkoelling/trust/compare/9089195213b6c2f45ae295d20196b0f87cea12ca...d4194468811485b83ac56656b4f1b691f7cdecae

It's a bit more fiddling to get it right, but I thought it was worth it to "communicate clearly".

**Comment by CCanerT** [comment]
@jkoelling that sounds good, but I dont think I can handle that easily since I am not that knowledgeable on programming. Maybe @osaatcioglu or @yusufff can do that. Meanwhile, @osaatcioglu I will check again and give you some info tomorrow. If I come up with something good as replacement translation, I also add that too.

**Comment by CCanerT** [comment]
For opening part "--Birinci dünya savaşı sırasında siperlerin içindeki askerler için barış ansızın geldi. 1914 Noelinde, "düşmanı görüldüğünüz yerde vurun" emrine karşın gene de-- İngiliz ve Alman askerleri siperlerini bırakıp, savaş alanını geçip ölülerini gömmek, hediyeler değiş-tokuşu yapmak ve oyun oynamak için tarafsız bölgede toplandılar.  --Öte yandan günümüzde batıda uzun süredir var olan bir barış ortamı olmasına rağmen gene de güven ortamı daha kötüye gidiyor.-- Araştırmalar gösteriyor ki son 40 yıl içerisinde her geçen gün insanlar daha az güveniyor. Yani, çelişki şurada .....

**Comment by CCanerT** [comment]
For the  POSSIBLE WIN-WINS " İki tarafın da bir şekilde kar etiği bir oyun oynamak zorundasın. Bir taraf zarar ederken diğer tarafın kar ettiği durumlar işbirliği için yararlı olmuyor. "

**Comment by CCanerT** [comment]
The rest is fairly understandable, so no need for tinkering.

**Comment by osaatcioglu** [comment]
@jkoelling great suggestion. But, I checked your code and it looks like I will need to modifications in several place and I am afraid that it will be changed too much from the actual code. Also, I have also finalized the translation so I don't need the extra space anymore. :) Thanks.

Thanks @CCanerT . I will review your suggestions and update accordingly.

**Comment by jkoelling** [comment]
@osaatcioglu you're welcome anyway! Yes, you'd have to modify the code wherever you need more space. It might look like a lot, but it's always the same increasing of the y-position (apart from the splash). Without changing the layout completely to something that flows by itself, there's no way around this. If you don't need it to fit the text, there's obviously no reason for it.
